Makefile

```diff
@@ -132,7 +132,7 @@ test/arg.beam: test/should_fail/arg.erl
 test_data_erls = $(wildcard test/known_problems/**/*.erl test/should_fail/*.erl test/should_pass/*.erl)
 build_test_data:
 	mkdir -p "test_data"
-	erlc -o test_data $(test_data_erls)
+	erlc $(ERLC_OPTS) -o test_data $(test_data_erls)
```

Comment:
The idea of not using the same `-I include -pa ebin` for the test programs as for the Gradualizer src itself was to make them independent. I think that was a good thing.

Is it the include or the ebin you need in the tests here?

Reply:
Actually, the `debug_info`, as I think it's necessary to refer to remote types from `test_module` as in:

```
gradualizer -pa ./test_data -- test/should_fail/<test_module>.erl
```

Reply:
Ah, so can we have a `TEST_ERLC_OPTS` instead or simply add `debug_info` on this command line?

Reply:
Yep. Ok, I'll take care of that 👍